platformcomponents/desktop/attachment-file-default.json

```diff
@@ -0,0 +1,21 @@
+{
+  "attachment-file-default": {
+    "sent": {
+      "comment": "Applied to sent & safe file attachments",
+      "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=5428%3A29",
+      "border": "@theme-outline-secondary-normal",
+      "iconForeground": "@theme-common-text-white",
```

Comment:
better to group these icon-box tokens like this
```
"previewbox":{
     "icon": "@theme-common-text-white",
     "background": "@theme-background-accent-normal"
}
```

Reply:
Pushed commit
`Grouped icon colors under previewBox key in attachment-file-default`
Changing this conversation to Resolved.